plasmapy/plasma/Lundquist_1D.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import scipy.special
+
+
+class Lundquist:
```

Comment:
I'm wondering about what we should name the class since "Lundquist" comes up in a couple of other contexts in plasma physics, like the Lundquist number.  What would you think of `ForceFreeFluxRope`?  

```suggestion
class ForceFreeFluxRope:
```